src/site/content/en/blog/kapwing/index.md

```diff
@@ -0,0 +1,427 @@
+---
+title: 'Kapwing: Powerful video editing for the web'
+subhead: |
+   Creators can now edit high-quality video content on the web with Kapwing, thanks to powerful APIs (like IndexedDB and WebCodecs) and performance tools.
+authors:
+  - joshuagrossberg
+date: 2021-12-06
+hero: image/VbsHyyQopiec0718rMq2kTE1hke2/to1mQCrluLRKhX5k97lc.jpeg
+alt: Kapwing logo
+description: |
+  Companies like Kapwing make it possible to create all this video content right on the web, by using powerful APIs (like IndexedDB and WebCodecs) and performance tools. 
+tags:
+  - blog
+  - case-study
+  - capabilities
+  - progressive-web-apps
+---
+
```

Comment:
@heyawhite 

Add an aside that will introduce Josh, like I did here for Chris https://web.dev/telegraph/.

Here's Josh's LinkedIn https://www.linkedin.com/in/joshua-grossberg-148b7a14

Add that Linkedin link to his author profile as well, so readers could reach out to him directly with questions (or comments).

cc @malchata for another tip for working on case studies

Reply:
His LinkedIn is in the profile -- not sure why it isn't showing up in the preview. Intro forthcoming

Reply:
Figured it out -- it was a formatting error (for a few authors, actually). Should be fixed. Got partnerships' LGTM, so now I just need one from @mihajlija or @malchata to publish.